openwisp_notifications/api/urls.py

```diff
@@ -9,42 +9,46 @@ def get_api_urls(api_views=None):
     if not api_views:
         api_views = views
     return [
-        path('', views.notifications_list, name='notifications_list'),
-        path('read/', views.notifications_read_all, name='notifications_read_all'),
-        path('<uuid:pk>/', views.notification_detail, name='notification_detail'),
+        path('notification/', views.notifications_list, name='notifications_list'),
         path(
-            '<uuid:pk>/redirect/',
-            views.notification_read_redirect,
-            name='notification_read_redirect',
+            'notification/read/',
+            views.notifications_read_all,
+            name='notifications_read_all',
         ),
         path(
-            'user-setting/',
-            views.notification_setting_list,
-            name='notification_setting_list',
+            'notification/<uuid:pk>/',
+            views.notification_detail,
+            name='notification_detail',
         ),
         path(
-            'user-setting/<uuid:pk>/',
-            views.notification_setting,
-            name='notification_setting',
+            'notification/<uuid:pk>/redirect/',
+            views.notification_read_redirect,
+            name='notification_read_redirect',
         ),
+        # WIP
+        # path(
+        #     'user/user-setting/',
+        #     views.notification_setting_list,
+        #     name='notification_setting_list',
+        # ),
+        # path(
+        #     'user/user-setting/<uuid:pk>/',
+        #     views.notification_setting,
+        #     name='notification_setting',
+        # ),
```

Comment:
I didn't mean to comment out these URLs, we should just flag them as deprecated in the docs.

Reply:
Yeah had already reverted it back.